images/structure.md

```diff
@@ -43,3 +43,29 @@ USER coder
 Your users can then create descendant images. These contain all of the original
 tooling and configuration installed onto the base image and the new
 customizations added by the users.
+
+## Coder assets
+
+Coder inserts assets into each workspace to enable code-server, JetBrains IDEs,
+and the Coder CLI. These assets are installed into the `/var/tmp/coder` directory
+of each workspace.
+
+You do not need to include these assets in your own custom images. However, when
+building your own images, the following utilities are required:
+
+- [POSIX Utilities](https://pubs.opengroup.org/onlinepubs/9699919799/idx/utilities.html)
+- [GNU libc](https://www.gnu.org/software/libc/libc.html)
+  - The minimum GNU libc version supported for the Coder-inserted assets is `2.1`
+  - Alpine is not supported, since it uses musl libc
+- [GNU Core Utilities](https://en.wikipedia.org/wiki/List_of_GNU_Core_Utilities_commands)
```

Comment:
I'm unsure if we are comfortable with referencing Wikipedia on our docs page. But this was the best listing of the utilities I could find.